lib/hammer_cli_katello/cv_import_export_helper.rb

```diff
@@ -28,7 +28,17 @@ def untar_export(options)
       export_tar_prefix = options[:prefix]
 
       Dir.chdir(export_tar_dir) do
-        `tar -xf #{export_tar_file}`
+        # Apache uid is always 48
+        if File.stat("#{export_tar_dir}/#{export_tar_file}").uid == 48
```

Comment:
What is `48`? (Yes, I've seen the comment above) ;-)
Please make this a constant instead of a magic number.

Reply:
also, should we store `export_tar_fullpath = File.join(export_tar_dir, export_tar_file)` and use that in this section, instead of a mixture of `"#{export_tar_dir}/#{export_tar_file}"` (full) and `"#{export_tar_file}"` (relative)